README.md

```diff
@@ -17,6 +17,22 @@ To run the API locally you need first to build and run docker container:
 ### Docker Build Image
 
     docker build -t sanctuary/api .
+    
+### Publish the Image
+
+Locally:
+```
+aws ecr get-login-password --region ca-central-1 | docker login --username AWS --password-stdin 990864907642.dkr.ecr.ca-central-1.amazonaws.com
+docker tag <image_id> 990864907642.dkr.ecr.ca-central-1.amazonaws.com/sanctuary/api
```

Comment:
Is there a way to avoid needing to know the image ID in this and the server step? If so, this could be scripted.

Reply:
The image is pushed to ECR via the makefile - currently it's tagged as v0.0.1 - you should be able to get the image that way.  There should be no need to run docker tag.

If you want the image available with another tag, the Makefile should be updated to push that.  E.g. we could push latest, or the git hash as tags.

I originally tagged with a fixed version so that it would be easy to reference when pulling the image.